### ISSUE TITLE
Tests for 9.0.1 and possible version bumps?

### ISSUE THREAD

**Issue by gspia** [Lysxia/first-class-families#39]
Hi

I was doing changes to "fcf-containers" and started to think about updating and preparing for the newer versions. I'm not sure if 9.0.1 and base change would be enough here in this lib in my case to be able do test 9.0.1 at fcf-containers, thou.

(Further, 8.10.3 and 8.8.4)

br i

**Comment by Lysxia** [comment]
Sure, I can update tests and bump bounds once GHC 9.0 is out.

In terms of CI, I'm moving from Travis to Github Actions (ongoing #40). Would you happen to know how to track GHC HEAD with Github Actions?

**Comment by gspia** [comment]
Oh, I thought that it was already out (saw version 9.0.1 on the NixOs repos) and will happily wait for the official announcement, too. (I tried already to build with 9.0.1 but I think that it was the version bounds of some other libs that were the problem.)

About your q about ghc head: I found https://github.com/haskell/actions/issues/14 and the link in there that might answer your question. Other than that, I'm taking my baby-steps with github actions and probably will configure fcf-containers to use them too to learn.
